### PR TITLE
sentry added

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**
Added Sentry init for code.
Require to set SENTRY_DSN=http://f052891fa3614d28b3f64e06268ec112@monitoring.lunie.io:9000/3

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 const dev = process.env.NODE_ENV === `development`
 const stargate = process.env.STARGATE || `http://localhost:9071`
 const rpc = process.env.RPC || `localhost:26657`
+const SENTRY_DSN = process.env.SENTRY_DSN || ''
```
Comment (colw) [inline config.js]: You could keep the variable name in camelCase for consistency.

File: pending/aleksei_sentry_added [hunk 2]
```diff
@@ -0,0 +1 @@
+[Added] [#3048](https://github.com/cosmos/lunie/issues/3048) Sentry integration was added @iambeone
```
Comment (faboweb) [inline pending/aleksei_sentry_added]: This is the wrong path for the changelog. Did you run `yarn changelog`?

Comment (iambeone) [reply pending/aleksei_sentry_added]: Yes

Comment (faboweb) [reply pending/aleksei_sentry_added]: Fixed it for you. As you can see here https://github.com/luniehq/lunie/blob/develop/package.json#L11 the entry should be created in ./changes. Maybe there is an issue on Windows? 🤷‍♂

Comment (iambeone) [reply pending/aleksei_sentry_added]: But it works for extension Oo

**Pull request merged**